highfidelity/hifi
# Tutorial fixes

The tutorial changes do not need to be QA'd, the majority of these changes have been in our internal tests for awhile, and the small changes I have made I have tested myself. What should be tested is that the vive controller display is working as expected.

To test:

1. Run `HMD.requestShowHandControllers();` and ensure that the hand controllers show up, and look correct
1. Ensure that running `Messages.sendLocalMessage('Controller-Set-Part-Layer', JSON.stringify({ trigger: "highlight" }));` highlights the controllers trigger yellow
1. Run `HMD.requestHideHandControllers();` and ensure that the hand controllers show up, and look correct
